lib/aes.ex

```diff
@@ -86,6 +79,19 @@ defmodule Fields.AES do
   # """ # doc commented out because https://stackoverflow.com/q/45171024/1148249
   @spec get_key(number) :: String
   defp get_key(key_id) do
-    Enum.at(@keys, key_id)
+    Enum.at(fetch_keys(), key_id)
+  end
+
+  # Dependency injection of encryption keys to allow flexibility in tests and consumers.
+  # I don't think fetching the keys from the env is going to be too slow, but
+  # consider optimizing if benchmarking shows it is.
```

Comment:
reading: http://stratus3d.com/blog/2020/09/30/performance-of-elixirs-system-get-env-function/ ⏳

Reply:
Env Encryption Key fetching perf benchmark issue added to backlog: https://github.com/dwyl/fields/issues/85